### PR TITLE
Including all-resources target in full run of admin-cli tests

### LINKED ISSUE #22128: Expand admin-cli tests to include "all-resources" target in full (i.e "all") run
Currently, "all-resources" target is not a part of full run. 
Need to enable "all-resources" target to be a part of full run.

### PR DESCRIPTION
**Pull request by kokil12**
Fixes #22128 : Modifying build.xml to include "all-resources" target as a part of full run of admin-cli tests.
Also, commenting out the tests that are failing currently on running "all-resources" target.

### CODE REVIEW

**Comment by kokil12** [comment]
@glassfishrobot Run CI tests please

**Comment by h2002044** [comment]
Looks fine. Go ahead and checkin. Please investigate the root cause for failure of commented out test later.

File: appserver/tests/appserv-tests/devtests/admin/cli/src/admin/AdminObjectTest.java [hunk 1]
```diff
@@ -371,7 +371,7 @@ private void testDeleteAdminObjectInStandaloneInstance() {
         reportResultStatus(testName, result);
         //reportExpectedResult(testName, result, STANDALONE_INSTANCE_NAME, INSTANCE1_NAME, INSTANCE2_NAME);
         reportExpectedResult(testName, result, "Administered object resource-1 deleted.");
-        reportUnexpectedResult(testName, result, SERVER, CLUSTER_NAME);
+        //reportUnexpectedResult(testName, result, SERVER, CLUSTER_NAME);
```
Comment (yaminikb) [inline appserver/tests/appserv-tests/devtests/admin/cli/src/admin/AdminObjectTest.java]: Please add a comment with the issue number that Deepak raised for this problem. Do that for all of the commented tests.

**Review by yaminikb — APPROVED** [review]

**Comment by kokil12** [comment]
@glassfishrobot Run CI tests please

**Pull request merged**